theora_image_transport/CMakeLists.txt

```diff
@@ -47,75 +46,74 @@ add_library(
   src/theora_subscriber.cpp
   src/manifest.cpp
 )
-
-include_directories(include
-  ${OpenCV_INCLUDE_DIRS}
-  ${PC_OGG_INCLUDE_DIRS}
-  ${PC_THEORA_INCLUDE_DIRS}
-  ${PC_THEORAENC_INCLUDE_DIRS}
-  ${PC_THEORADEC_INCLUDE_DIRS})
-
+target_compile_definitions(${LIBRARY_NAME} PRIVATE
+  ${PC_OGG_CFLAGS_OTHER}
+  ${PC_THEORA_CFLAGS_OTHER}
+  ${PC_THEORAENC_CFLAGS_OTHER}
+  ${PC_THEORADEC_CFLAGS_OTHER}
+)
+target_include_directories(${LIBRARY_NAME} PRIVATE
+  "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+  "$<INSTALL_INTERFACE:include/${PROJECT_NAME}>"
+)
 target_link_libraries(${LIBRARY_NAME}
   ${OpenCV_LIBRARIES}
```

Comment:
Ni, I think this could be just the target `opencv_imgproc`

Example
https://github.com/ros-perception/vision_opencv/blob/0b69de38953f21090ac62bc90eb700e8bc98a023/cv_bridge/src/CMakeLists.txt#L12

Reply:
Fixed in ecf35d4.

Reply:
Looks like `${OpenCV_LIBRARIES}` didn't get deleted.

Reply:
Bah, sorry.  Removed in 0c14e91